src/server.mjs

```diff
@@ -113,6 +114,36 @@ var limits = {
 };
 var upload = multer({ storage, limits });
 
+// runs every hour
+// deletes any files in the download directory past the set fileExpirationTime set in config
+cron.schedule('0 * * * *', () => {
+  console.log("cron scheduled check");
+  const currentTime = new Date().getTime();
+  // loop through these specified directories
+  for (const dir of [DOWNLOAD_DATA_PATH, UPLOAD_DATA_PATH]) {
```

Comment:
Should we maybe warn people near where these are configured that the tube map server needs to use them exclusively, and will delete anything else in there?